.github/workflows/smoke-test.yml

```diff
@@ -71,8 +71,9 @@ jobs:
               psql -c "INSERT INTO test_geometry_table (geom) VALUES (ST_GeomFromText('POINT(0 0)', 4326));"
               psql -c "SELECT * FROM test_geometry_table;"
 
+              echo "Test zombodb Extension"
               psql -c "CREATE EXTENSION zombodb;"
-              psql -c "SELECT * FROM pg_extension WHERE extname = 'zombodb';"
+              psql -c "SELECT zdb.internal_version();"
```

Comment:
`psql -c "SELECT * FROM pg_extension WHERE extname = 'zombodb';" ` returns a row for each installed version of the extension.
 `psql -c "SELECT zdb.internal_version();"` returns the version of zombodb currently being used.

Reply:
`psql -c "SELECT * FROM pg_extension WHERE extname = 'zombodb';" ` will return 0 exit code even when there is no zombodb installed. So it will never fail in the smoke test. If we are only installing one version of ZomboDB, the output from both scenarios would be equivalent.